schema-based-on-diagram.sql

```diff
@@ -0,0 +1,41 @@
+CREATE DATABASE clinic;
+
+CREATE TABLE patients (
+  id              INT  PRIMARY KET GENERATED ALWAYS AS IDENTITY,
+  name            CHAR(100) NOT NULL, 
+  date_of_birth   DATE NOT NULL
```

Comment:
- [ ] Kindly make use of the data type provided in the ERD diagram which is `VARCHAR` with of course the string length validation instead of using `CHAR`. This is because the two have differences in performance: a `CHAR` column is always padded to the defined length. So if you define a column as `CHAR(100)` and one as `VARCHAR(100)` but only store 10 characters in each, the `CHAR(100)` column uses 100 characters for each value (the 10 characters you stored, plus 90 spaces), whereas the `VARCHAR` column only stores 10 characters. So to avoid this unnecessary padding, let's make use of `VARCHAR`. You can read more on this [here](https://dba.stackexchange.com/questions/126003/index-performance-for-char-vs-varchar-postgres). Kindly make these changes in all the places you are using `CHAR`